README.md

```diff
@@ -27,19 +27,23 @@ Install the latest version of `confluent` to `/usr/local/bin`:
 
     brew install confluentinc/tap/cli
 
-#### Scripted installation
+#### APT (Ubuntu and Debian)
 
-Install the latest version of `confluent` to `/usr/local/bin`:
-
-    curl -sL https://raw.githubusercontent.com/confluentinc/cli/main/install.sh | sh -s -- -b /usr/local/bin
+Install the latest version of `confluent` to `/usr/bin` (requires `glibc 2.17` or above for `amd64` and `glibc 2.27` or above for `arm64`):
 
-Print a complete list of versions available for download:
+    wget -qO - https://packages.confluent.io/confluent-cli/deb/archive.key | sudo apt-key add -
+    sudo apt install software-properties-common
+    sudo add-apt-repository "deb https://packages.confluent.io/confluent-cli/deb stable main"
+    sudo apt update && sudo apt install confluent-cli
 
-    curl -sL https://raw.githubusercontent.com/confluentinc/cli/main/install.sh | sh -s -- -l
+#### YUM (RHEL and CentOS)
 
-Install `confluent` v3.6.0 to `/usr/local/bin`:
+Install the latest version of `confluent` to `/usr/bin` (requires `glibc 2.17` or above for `amd64` and `glibc 2.27` or above for `arm64`):
```

Comment:
Would it be more direct to include rhel and centos version numbers here? or i guess users will need to look up the glibc version they're using

Reply:
In practice I don't think it will matter much, because current Linux distros are well above both of these. So I included the bit about glibc as an attempt to prevent questions about it if someone using an ancient distribution runs into problems.